client/widgets/VideoWidget.cpp

```diff
@@ -33,11 +36,28 @@ VideoWidgetBase::~VideoWidgetBase() = default;
 
 void VideoWidgetBase::playVideo(const VideoPath & fileToPlay)
 {
+	OBJECT_CONSTRUCTION;
+
+	using SubTitlePath = ResourcePathTempl<EResType::SUBTITLE>;
+	SubTitlePath subTitlePath = fileToPlay.toType<EResType::SUBTITLE>();
+	SubTitlePath subTitlePathVideoDir = subTitlePath.addPrefix("VIDEO/");
+	if(CResourceHandler::get()->existsResource(subTitlePath))
+	{
+		auto rawData = CResourceHandler::get()->load(subTitlePath)->readAll();
+		srtContent = std::string(reinterpret_cast<char *>(rawData.first.get()), rawData.second);
```

Comment:
what about encoding?

Reply:
Haven't tested it yet with non ASCII stuff. How to handle it correctly as utf-8 in C++?